update-third-party-libs.sh

```diff
@@ -14,7 +14,7 @@
 
 (cd both/components/; wget -N http://www.jflei.com/jChester/jChester.js)
 
-(cd both/components/; curl -s https://raw.githubusercontent.com/OpenBookPrices/country-data/master/data/countries.json | sed -e '1s/^/countries = /' -e '$a\;' > countries.js)
+(cd both/components/; curl -s https://raw.githubusercontent.com/OpenBookPrices/country-data/master/data/countries.json | sed -e '1s/^/JSON.stringify(/' -e '$a\,function(k,v,u){if(k!=="name"&&k!=="alpha2"&&isNaN(k)){return u}return v},2)' | node -p | sed -e '1s/^/countries = /' -e '$a\;' > countries.js)
```

Comment:
LOL, that's cute. This isn't a shortest code competition, though, lets go with just 2 arguments and undefined instead, please. You should get some extra characters from switching to a ternary operator anyways =).

When is the key an integer? Is this a convention I'm not familiar with?